src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,31 @@
+.catalog {
+  padding: 50px 40px;
+}
+
+.grid {
+  display: grid;
+
+  grid-template-rows: repeat(2, 408px);
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border: 2px solid #000;
+  background-color: yellow;
+}
```

Comment:
This looks weird, as you already have `.card` defined in `card.scss`
Please merge these styles.